### ISSUE TITLE
Add aria-labelledby to Find Your Institution

### ISSUE THREAD

**Issue by fredrikdomeij** [TheIdentitySelector/thiss-js#223]
**Find Your Institution** and **Choose Your Institution** should have the attribute **aria-labelledby** (or some more useful attribute) so that screen readers know the important thing to notify the user of on page load and when selecting the **searchinput** search field.

**Comment by Bojhan** [comment]
Fix is merged.
